src/main/java/ch/wisv/areafiftylan/security/TicketTransferToken.java

```diff
@@ -0,0 +1,39 @@
+package ch.wisv.areafiftylan.security;
+
+import ch.wisv.areafiftylan.model.Ticket;
+import ch.wisv.areafiftylan.model.User;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+
+/**
+ * Created by beer on 11-3-16.
+ */
+@Entity
+public class TicketTransferToken extends Token {
+
+    @OneToOne(targetEntity = Ticket.class, fetch = FetchType.EAGER)
+    @JoinColumn(nullable = false)
+    private Ticket ticket;
+
+    @OneToOne(targetEntity = User.class, fetch = FetchType.EAGER)
+    @JoinColumn(nullable = false)
+    private User goalUser;
```

Comment:
You can remove this and just use the `Token` user as goal user. We don't need to store the Ticket sender, as this is the Ticket owner